lib/rubocop/cop/lint/ambiguous_regexp_literal.rb

```diff
@@ -61,6 +61,8 @@ def find_offense_node(node, regexp_receiver)
         end
 
         def method_chain_to_regexp_receiver?(node)
+          return false unless node.parent.receiver
```

Comment:
It seem `node.parent` is not repeated 3 times in the code. Perhaps a local variable is in order.

Reply:
Sure. I updated it!